docs/renderer.md

```diff
@@ -0,0 +1,32 @@
+# MathJax Renderer Definitions
+
+When math input is expected from students, edX offers a math preview that attempts to show their expression in normal mathematical notation. This preview is rendered by MathJax via AsciiMath. While the preview does a reasonably good job, there are many situations where it falls down, even for standard edX functions (for example, try typing in `1/arctanh(x)` in an edX input box!). We have constructed a series of renderer definitions to supplement the standard AsciiMath definitions in order to provide better previews.
```

Comment:
Question: Does the preprocessor above work with `<textline math='true' />` AND `<formulaequationinput />`? How much of this discussion is specific to `<textline math='true' />`? For example, `<formulaequationinput />` displays `1/arctanh(x)` correctly. But it would be great if `<formulaequationinput />` could get the factorial improvements, too.

I know that MIT Physics tends to favor the `<textline />` approach, but I _believe_ Jen prefers `<formulaequationinput />`. And it seems likely that lots of people don't even know `<textline math='true' />` exists. So maybe nice to **briefly** mention the difference?

Reply:
Ah. `formulaequationinput` doesn't let us use vecx and hatx, so I've never used it. Now, `textline` uses AsciiMath, while `formulaequationinput` uses server-side parsing to generate LaTeX. So, all of these renderer things only apply to textline. I'll update the documentation appropriately.